code/modules/atmospherics/components/omni_devices/_omni_extras.dm

```diff
@@ -1,15 +1,10 @@
 //--------------------------------------------
 // Omni device port types
 //--------------------------------------------
-#define ATM_NONE	0
-#define ATM_INPUT	1
-#define ATM_OUTPUT	2
-
-#define ATM_O2		3
-#define ATM_N2		4
-#define ATM_CO2		5
-#define ATM_N2O		6
-#define ATM_H2		7
+#define ATM_NONE		0
+#define ATM_INPUT		1
+#define ATM_OUTPUT		2
+#define ATM_FILTER		8 //as to not interfer with legacy handling.
```

Comment:
Could you change these to use `BITFLAG(n)` while you're in the area?

Reply:
That seems like a bad idea. These are still manually set via mapping.

Reply:
Misunderstood these as bitflags due to `1` `2` `8`